crates/shared/src/http_solver_api/model.rs

```diff
@@ -139,6 +147,7 @@ pub struct SettledBatchAuctionModel {
     pub ref_token: Option<H160>,
     #[serde_as(as = "HashMap<_, DecimalU256>")]
     pub prices: HashMap<H160, U256>,
+    pub interaction_data: Option<Vec<InteractionData>>,
```

Comment:
Nicer to define as non optional and with `#[serde(default)]` so that it becomes empty Vec is field is missing.